DataFormats/DeepFormats/interface/ChargedCandidateFeatures.h

```diff
@@ -0,0 +1,41 @@
+#ifndef DataFormats_DeepFormats_ChargedCandidateFeatures_h
+#define DataFormats_DeepFormats_ChargedCandidateFeatures_h
+
+#include "DataFormats/Common/interface/CMS_CLASS_VERSION.h"
+
+namespace btagbtvdeep {
+
+class ChargedCandidateFeatures {
+
+  public:
+
+    float ptrel;
+    float puppiw;
+    float VTX_ass;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.

Reply:
will do

Reply:
fixed with https://github.com/cms-sw/cmssw/pull/19893/commits/347c96268c4f6f017384998cee2bdee31372fa64